content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_kubelet-config.md

```diff
@@ -0,0 +1,77 @@
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference conent, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+
+Atualize a configuração do kubelet para este nó
+
+### Sinopse
+
+Baixe a configuração do kubelet do kubelet-config ConfigMap armazenado no cluster
```

Comment:
This paragraph needs to be reviewed against the English version. It's missing more information that exist in the synopsis paragraph.